### PR TITLE
Add test check for scenario within error reporting

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/45890

### CODE REVIEW

File: lib/OpenQA/Schema/Result/Jobs.pm [hunk 1]
```diff
@@ -296,6 +293,17 @@ sub scenario_name {
     return $scenario;
 }
 
+# searches for the testsuite where the name matches our TEST setting ignoring "+..." suffix
```
Comment (okurz) [inline lib/OpenQA/Schema/Result/Jobs.pm]: But the "+…" suffix is just a convention, these are still individual testsuites. I am not sure this is what we want

Comment (Martchus) [reply lib/OpenQA/Schema/Result/Jobs.pm]: If that's not what we want, then I remove the first commit considering the current behavior is fine after all. (The 2nd one which adds an additional check is likely still useful.)

**Comment by Martchus** [comment]
Since the adjustment for the suffix is not wanted this leaves the remaining thing to be fixed the scenario name being rendered as `OpenQA::Schema::Result::TestSuites=HASH(0x9daa1d8)`. Not sure how to reproduce that since the test for it in the PR apparently doesn't.

**Comment by okurz** [comment]
right, will you exclude the commit then? I think it's still shown up to now

**Comment by Martchus** [comment]
I'll exclude it tomorrow (I've just been waiting for feedback before continuing.)

**Comment by Martchus** [comment]
I've merged this for the additional check in the tests. Debugging the problem which I could only see on OSD is an ongoing task...

**Pull request merged**